config/doctrine.php

```diff
@@ -171,5 +171,6 @@
     */
     'gedmo'                     => [
         'all_mappings' => false
-    ]
+    ],
+    'doctrine_presence_verifier' => true,
```

Comment:
Could you add a comment block on top of this with the explanation of what it does?